tools/internal_ci/linux/grpc_sample_fuzzers.sh

```diff
@@ -15,13 +15,23 @@
 
 set -ex
 
+# avoid slow finalization after the script has exited.
```

Comment:
just to help me understand - why are these fuzzers called "sample" fuzzers?

Reply:
the thing we're trying to get to here is to just run the fuzzers for a little bit... normally you'd run them for ~24 hours, but here we just want to... "sample" them to see if there's anything that they quickly show up with the new change